src/views/Books/BookList.js

```diff
@@ -18,13 +19,22 @@ function BookList() {
   if (loading) return <h1>Loading books...</h1>;
 
   return (
-    <ul className="book-list" aria-label="book list">
-      {books.map((book) => (
-        <li key={book.book_id}>
-          <Book book={book} />
-        </li>
-      ))}
-    </ul>
+    <>
+      <div>
+        <p>
+          Return to <a href="/">Welcome</a>
+        </p>
```

Comment:
Remember to use Link to prevent the page from reloading and refetching all of the data.